assets/monitoring/grafana/v1alpha1/grafanadashboards.go

```diff
@@ -9,10 +9,19 @@ import (
 	"io/fs"
 	"path/filepath"
 	"regexp"
+	"slices"
 )
 
 var (
 	grafanaDashboardsFileRegex = regexp.MustCompile(`^[^/]+\.json$`)
+
+	// Due to https://github.com/scylladb/scylla-operator/issues/2822 we need to exclude the following platform dashboards
+	// from the operator until the root cause is fixed. Currently, the Operator doesn't support ScyllaDB 2025.3 so it's safe to
+	// do so.
```

Comment:
Suggested rewording to make the intent clear and the problem actionable.

```suggestion
	// FIXME: remove the exclusions and fix https://github.com/scylladb/scylla-operator/issues/2822
	// before supporting ScyllaDB 2025.3.
```